detector/vuls2/db.go

```diff
@@ -0,0 +1,189 @@
+package vuls2
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/future-architect/vuls/logging"
+	"github.com/klauspost/compress/zstd"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+	progressbar "github.com/schollz/progressbar/v3"
+	"golang.org/x/xerrors"
+	oras "oras.land/oras-go/v2"
+	"oras.land/oras-go/v2/content"
+	"oras.land/oras-go/v2/content/memory"
+	"oras.land/oras-go/v2/registry/remote"
+
+	db "github.com/MaineK00n/vuls2/pkg/db/common"
+	utilos "github.com/MaineK00n/vuls2/pkg/util/os"
+)
+
+const (
+	SchemaVersion = 0
+	dbMediaType   = "application/vnd.vulsio.vuls.db.layer.v1+zstd"
+)
+
+var (
+	// DefaultGHCRRepository is GitHub Container Registry for vuls2 db for vuls0
+	DefaultGHCRRepository = fmt.Sprintf("%s:%d", "ghcr.io/vulsio/vuls-nightly-db", SchemaVersion)
+
+	// DefaultPath is the path for vuls2 db file
+	DefaultPath = filepath.Join(utilos.UserCacheDir(), "vuls.db")
```

Comment:
Set default path to CWD/vuls.db

Reply:
https://github.com/future-architect/vuls/compare/5c54a7d8db4e217a887e72462d3d0016a747458e..15866221afd8e9ecfe42b36392ed2296ddd70f23